src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -4184,14 +4184,14 @@ private LinkedHashMap<InetAddressAndPort, Float> getEffectiveOwnership(String ke
         if (keyspace != null)
         {
             if (isLocalSystemKeyspace(keyspace))
-                throw new IllegalStateException("Ownership values for keyspaces with LocalStrategy are meaningless");
+                throw new IllegalStateException("Ownership values for keyspaces " + keyspace + " with LocalStrategy are meaningless");
```

Comment:
@maoling I do not think this is necessary. This code path is exercised when `keyspace != null` so a user knows which keyspace it is talking about, no?